.travis.yml

```diff
@@ -24,6 +24,8 @@ jobs:
         - python3 codalab_service.py start --services default monitor --version ${TRAVIS_BRANCH}
       script:
         - python3 codalab_service.py test default --version ${TRAVIS_BRANCH}
+          # Make sure restarting worker doesn't cause any issues (ie in serialization/deserialization
+        - docker stop codalab_worker_1 && python3 codalab_service.py start --services worker --version ${TRAVIS_BRANCH} && python3 codalab_service.py test run --version ${TRAVIS_BRANCH}
```

Comment:
Maybe running the `run` tests again isn't necessary.

Reply:
What about just doing `docker restart codalab_worker_1`?

I guess you need to do something to test that the worker is alive and well...